assets/data-labeling/environments/data-labeling-sam/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,28 @@
+name: data-labeling-sam
+channels:
+- conda-forge
+- pytorch
+dependencies:
+- python=3.9
+- pip=21.3.1
+- pip:
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-contrib-dataset=={{latest-pypi-version}}
+  - azureml-dataset-runtime[fuse,pandas]
+  - azureml-defaults=={{latest-pypi-version}}
+  - azure-ml==0.0.1
+  - azure-ml-component==0.9.18.post2
+  - azureml-mlflow==1.51.0
+  - azureml-contrib-services==1.51.0
+  - azureml-contrib-services==1.51.0
+  - torch-tb-profiler~=0.4.0
+  - azureml-automl-common-tools==1.51.0
```

Comment:
Looks like this needs to change to 1.52.0. See the [build logs](https://github.com/Azure/azureml-assets/actions/runs/5664653344/job/15348256628?pr=925#step:7:515).